### ISSUE TITLE
Data Sceince

### ISSUE THREAD

**Issue by tanviagwl98** [workattech/hacktoberfest-2020#26]
Is this repository accepting **Data Science** articles too? I have an article on Cartoonization of Images using simple python script.
Please let me know if I can add that here.

**Comment by gcnit** [comment]
If it is already written, please feel free to submit

**Comment by tanviagwl98** [comment]
Yes,  I have published that on medium and have a valid code repository for that.  I will create a pull request soon.
